change/@azure-msal-browser-8ea0dcd6-f5bb-4106-b6ce-b7442e8848de.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Patch?

Reply:
We're forcing a minor bump to catch up to 1p